app/screens/ProductDetailsScreen/ProductDetailsScreen.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Image, Text, SafeAreaView, ScrollView, Button, Alert, RefreshControl } from 'react-native';
+import { Header } from '../../components/Header/Header';
+import { IconBasket } from '../../components/icons/IconBasket';
+import { IconHeart } from '../../components/icons/IconHeart';
+import { IconArrowLeft } from '../../components/icons/IconArrowLeft';
+
+export const ProductDetailsScreen: React.FC = (): JSX.Element => {
+    const [refreshing, setRefreshing] = useState(false);
+    const product = {
+        attributes: {
+            price: '86.99',
+            description: 'Doloremque ab accusamus blanditiis et. Deleniti ma…oloribus ad. Non repellat maiores delectus rerum.',
+            name: 'Raw Edge T Shirt'
+        },
+        id: '16'
+    };
+
+    const wait = (timeout: number) => {
+        return new Promise(resolve => setTimeout(resolve, timeout));
+    }
+
+    const onRefresh = React.useCallback(() => {
+        setRefreshing(true);
+        wait(2000).then(() => setRefreshing(false));
+    }, []);
+
+    return (
+    <SafeAreaView>
+        <ScrollView
```

Comment:
wrong position of scrollview. It should wrap only scrolable content. Header must persist for user to be able at least to return back at any time